application/config/internal.php

```diff
@@ -118,7 +118,7 @@
         // These are defaults and are later overwritten in LSYii_Application by a path based on config tempdir/tempurl
         'assetManager' => array(
             'excludeFiles' => array("config.xml", "assessment.pstpl", "clearall.pstpl",  "completed.pstpl",  "endgroup.pstpl",  "endpage.pstpl",  "groupdescription.pstpl",  "load.pstpl",  "navigator.pstpl",  "printanswers.pstpl",  "print_group.pstpl",  "print_question.pstpl",  "print_survey.pstpl",  "privacy.pstpl",  "question.pstpl",  "register.pstpl",  "save.pstpl",  "startgroup.pstpl",  "startpage.pstpl",  "surveylist.pstpl",  "survey.pstpl",  "welcome.pstpl" ),
-            'baseUrl' => '/tmp/assets',
+            'baseUrl' => '/lime-dev/tmp/assets',
```

Comment:
Resetted here : https://github.com/LimeSurvey/LimeSurvey/blob/master/application/core/LSYii_Application.php#L90 in 2.XX

Maybe i lost some part from internal (set asseturl to null here : https://github.com/LimeSurvey/LimeSurvey/commit/1878a118eccee076e9b986ae871a1cafc1973c42#diff-f5392477bcb7ace25c65e8bd7abd39e9R67)